### DISCUSSION TITLE
Backstage plugin

### DISCUSSION THREAD

**Discussion by jbadeau** [IKatsuba/nx-cloud#229] [Announcements]
Hello,

I am planning on creating a backstage.io plugin to display Nx build logs, cache-hits, etcs.

I have a few questions before I begin:

- Are you using the offical nx-cloud runner api?
- What are the @nx-turbo/* modules and where do they come from? I cant find the source.
- Are you capturing everything the nx-cloud runner sends, i.e. logs, cache hit/miss?

**Comment by IKatsuba** [comment]
Hi @jbadeau!

1. I am using the official NX Cloud runner. This project does not have its runner.
2. Take a look at the lib directory. There are all sources.
3. I do it on the backend

**Comment by tudi2d** [comment]
Great to see @jbadeau! If you feel ready feel free to add your plugin to our [Backstage.io plugin marketplace](https://backstage.io/plugins/) to let others discover it easily. Also a heads up that we are aiming for creating plugins as seperate repositories outside of the `backstage/backstage` repository with [this RFC](https://github.com/backstage/backstage/issues/20266).

Have fun coding & looking forward to the plugin! :)
